### PR TITLE
NIC 60k upgrade inventory

### PR DESCRIPTION
**Pull request by sravfeyn**
@javierwilson @calellowitz @emord @snopoke 

Don't merge yet

### CODE REVIEW

File: fab/inventory/icds [hunk 1]
```diff
@@ -72,28 +109,35 @@ kafka0
 
 [celery:children]
 celery0
+celery1
 
 [pillowtop:children]
 pil0
```
Comment (sravfeyn) [inline fab/inventory/icds]: We will need to add `pil1` here after we update HQ to support multiple pillow machines. We will also need to do same for kafka and add `pil1` to `kafka:children`

File: fab/inventory/icds [hunk 2]
```diff
@@ -72,28 +109,35 @@ kafka0
...
 
 [touchforms:children]
-pil0
+pil1
```
Comment (sravfeyn) [inline fab/inventory/icds:120]: Should we take any special care when moving touchforms over while doing ansible deploy?

File: fab/inventory/icds [hunk 3]
```diff
@@ -39,6 +39,37 @@
 [control]
 10.247.24.20
 
+[pg2]
+10.247.24.22 devices='["/dev/sdb"]' partitions='["/dev/sdb1"]' datavol_device='/dev/mapper/consolidated-data' hot_standby_server='10.247.24.28'
```
Comment (sravfeyn) [inline fab/inventory/icds:43]: - [ ] To update actual devices after partition or don't need it if doing without extra storage

File: fab/inventory/icds [hunk 4]
```diff
@@ -48,11 +79,17 @@ web0
...
+web3
+web4
+web5
 kafka0
 
 [postgresql:children]
 pg0
 pg1
+pg2
+pg3
```
Comment (sravfeyn) [inline fab/inventory/icds:92]: How should the shards be updated [here](https://github.com/dimagi/commcarehq-ansible/blob/master/ansible/vars/icds/icds_public.yml#L81)?

File: fab/inventory/icds (hunk 2 above)
Comment (benrudolph) [reply fab/inventory/icds:120]: touchforms is stateless so it shouldn't matter

**Comment by emord** [comment]
I think you guys will also want to update the celery queues https://github.com/dimagi/commcare-hq-deploy/blob/master/fab/environments.yml#L67

**Comment by emord** [comment]
@sravfeyn can this be closed out? I think all of these have been deployed at some point

**Comment by dannyroberts** [comment]
closing

**Pull request closed without merging**